File: backend/Functions/Edna.Platforms/Edna.Platforms/PlatformDto.cs
using IdentityModel.Jwk;

namespace Edna.Platforms
{
    public class PlatformDto
    {
        public string Id { get; set; }
        public string DisplayName { get; set; }
        public string Issuer { get; set; }
        public string JwkSetUrl { get; set; }
        public string AccessTokenUrl { get; set; }
        public string AuthorizationUrl { get; set; }
        public string LoginUrl { get; set; }
        public string LaunchUrl { get; set; }
        public string ClientId { get; set; }
        public string PublicKey { get; set; }
        public string InstitutionName { get; set; }
        public string LogoUrl { get; set; }
        public string ToolJwkSetUrl { get; set; }
        public string ToolJwk{get; set;}


    }
}

File: client/src/Dtos/Platform.dto.ts
/*---------------------------------------------------------------------------------------------
 *  Copyright (c) Microsoft Corporation. All rights reserved.
 *  Licensed under the MIT License.
 *--------------------------------------------------------------------------------------------*/

export interface PlatformDto {
  displayName: string;
  issuer: string;
  jwkSetUrl: string;
  accessTokenUrl: string;
  authorizationUrl: string;
  loginUrl: string;
  launchUrl: string;
  domainUrl: string;
  clientId: string;
  institutionName: string;
  logoUrl: string;
  publicKey: string;
  toolJwk: string;
  toolJwkSetUrl: string;
}
